CONTRIBUTING.md

```diff
@@ -262,6 +262,125 @@ or
 
 ``./test_infra/scripts/delete-stack.sh databases``
 
+## Ray Load Tests Environment 
+**DISCLAIMER**: Make sure you know what you are doing. These steps will charge some services on your AWS account and require a minimum security skill to keep your environment safe.
+
+* Pick up a Linux or MacOS.
+* Install Python 3.7, 3.8 or 3.9 with [poetry](https://github.com/python-poetry/poetry) for package management
+* Fork the AWS SDK for pandas repository and clone that into your development environment
+* Go to the project's directory create a Python's virtual environment for the project
+
+`python -m venv .venv && source .venv/bin/activate`
```

Comment:
Just realising that we should remove these mentions from everywhere in the `CONTRIBUTING.md` guide since we don't need a .venv given that poetry creates one for us

Reply:
Good catch